### PR TITLE
frontend grafana dashboards in docker

### PR DESCRIPTION
**Pull request by uwedeportivo**
this is connected to

sourcegraph/sourcegraph#5543

this is how this looks like in the docker deploy context

### CODE REVIEW

File: deploy-frontend-internal.sh [hunk 1]
```diff
@@ -27,6 +27,6 @@ docker run --detach \
     -e ZOEKT_HOST=zoekt-webserver:6070 \
     -e LSIF_SERVER_URL=http://lsif-server:3186 \
     -v ~/sourcegraph-docker/sourcegraph-frontend-internal-0-disk:/mnt/cache \
-    sourcegraph/frontend:3.7.1
+    sourcegraph/frontend:3.8-internal-uwe
```
Comment (slimsag) [inline deploy-frontend-internal.sh]:
```suggestion
    sourcegraph/frontend:3.7.1
```

File: deploy-frontend.sh [hunk 2]
```diff
@@ -27,9 +27,10 @@ docker run --detach \
     -e REPO_UPDATER_URL=http://repo-updater:3182 \
     -e ZOEKT_HOST=zoekt-webserver:6070 \
     -e LSIF_SERVER_URL=http://lsif-server:3186 \
+    -e GRAFANA_SERVER_URL=htto://grafana:3000 \
     -v ~/sourcegraph-docker/sourcegraph-frontend-$1-disk:/mnt/cache \
     -p 0.0.0.0:$((3080 + $1)):3080 \
-    sourcegraph/frontend:3.7.1
+    sourcegraph/frontend:3.8-internal-uwe
```
Comment (slimsag) [inline deploy-frontend.sh]:
```suggestion
    sourcegraph/frontend:3.7.1
```

File: deploy-grafana.sh [hunk 3]
```diff
@@ -19,12 +19,12 @@ docker run --detach \
...
     -v $(pwd)/grafana:/sg_config_grafana/provisioning/datasources \
     -e GF_AUTH_ANONYMOUS_ENABLED=true \
-    -e GF_AUTH_ANONYMOUS_ORG_NAME=Sourcegraph \
+    -e GF_AUTH_ANONYMOUS_ORG_NAME='Main Org.' \
     -e GF_AUTH_ANONYMOUS_ORG_ROLE=Editor \
     -e GF_USERS_ALLOW_SIGN_UP='false' \
     -e GF_USERS_AUTO_ASSIGN_ORG='true' \
     -e GF_USERS_AUTO_ASSIGN_ORG_ROLE=Editor \
-    sourcegraph/grafana:6.3.3@sha256:2f68b9b1542e7d75459d983b606d2fdd1c11a75610464e3d7a6ced4f3ac474bf
+    sourcegraph/grafana:6.3.3-1-internal-uwe
```
Comment (slimsag) [inline deploy-grafana.sh]:
```suggestion
    sourcegraph/grafana:6.3.3-2@sha256:something
```

**Comment by uwedeportivo** [comment]
re: -internal-uwe. of course not :-)

**Pull request merged**